<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>600</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="doubleWindow">
     <attribute name="title">
      <string>双串口</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="mcuWindow">
     <attribute name="title">
      <string>模拟V4MCU</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="toolWindow">
     <attribute name="title">
      <string>小工具</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="aboutWindow">
     <attribute name="title">
      <string>关于</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
